app/src/main/java/org/jellyfin/androidtv/base/IItemClickListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.jellyfin.androidtv.base
+
+import androidx.leanback.widget.Presenter
```

Comment:
We don't need this import. When using Android Studio you can have it fix your imports when creating commits. (option for this is in the commit dialog)

Reply:
Okay, setting that now for the future, and removing this import. Thanks :)